esp/esp/qsd/views.py

```diff
@@ -179,7 +179,7 @@ def qsd(request, url):
 
         # Enforce authorizations (FIXME: SHOW A REAL ERROR!)
         if not have_edit:
-            raise ESPError("You don't have permission to edit this page.", log=False)
+            raise Http403, "You don't have permission to edit this page."
```

Comment:
It does, in fact as far as I can tell they're exactly equivalent.

I only did it this way because the rest of the file does it that way; it's probably better to actually update all of them to the other way to avoid mixup with `raise (Http403, "message")`, but I didn't feel like updating the other uses of it.